### ISSUE TITLE
Upload new file issue

### ISSUE THREAD

**Issue by Issen007** [django-wiki/django-wiki#1015]
Hi,
I got problem with Django-Wiki to upload files.

When I upload a file it accept the file type but when it redirect to next page I got the website is not availbile but it points to url `http://localhost:8000/_plugin/attachments/`

And in the console it only tells me following information
```
GET /_plugin/attachments/ HTTP/1.1" 200 11907
"GET /favicon.ico/ HTTP/1.1" 302 0
"GET /_create/?slug=favicon.ico HTTP/1.1" 200 8289
"GET /notifications/json/get/0/?_=1579276244438 HTTP/1.1" 200 50
"POST /_plugin/attachments/ HTTP/1.1" 200 12136
"GET /favicon.ico/ HTTP/1.1" 302 0
"GET /_create/?slug=favicon.ico HTTP/1.1" 200 8289
"GET /notifications/json/get/0/?_=1579276256608 HTTP/1.1" 200 50
"POST /_plugin/attachments/ HTTP/1.1" 200 12039
``` 

Any help you be great
thx

**Comment by Issen007** [comment]
To just give out some more information.
I have create a new Django instance and installed via `pip install`

First I though it was the `media` part in the urls.py that was missing but I still got the same issue.

**urls.py**
```
from django.contrib import admin
from django.conf.urls.static import static
from django.conf import settings
from django.urls import path, include

urlpatterns = [
    path('admin/', admin.site.urls),
    path('notifications/', include('django_nyt.urls')),
    path('', include('wiki.urls'))
]

urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

part of **settings.py**
```
STATIC_URL = '/static/'
STATIC_ROOT = os.path.join(BASE_DIR, 'static_cdn')
STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static')]

MEDIA_URL = '/media/'
MEDIA_ROOT = os.path.join(BASE_DIR, 'media')

LOGIN_URL = '/account/signin'
```

**Comment by benjaoming** [comment]
Thanks for the report, @Issen007 - I was wondering if you have tried this from the latest `master` branch? It will be released soon.

You can install it with `pip install https://github.com/django-wiki/django-wiki/archive/master.zip`

**Comment by Issen007** [comment]
@benjaoming thanks for the advice and upgrading from the github solve the issue. 
But we need to upgrade the pip installation also, the version 0.5 has the issue but I guess that have been solved in 0.6
